tests/portstat/test_portstat.py

```diff
@@ -84,6 +84,15 @@ def parse_portstat(content_lines):
 
     return results
 
+@pytest.fixture(scope='module', autouse=True)
+def enable_counters(duthosts, enum_rand_one_per_hwsku_frontend_hostname):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    logger.info('Enabling all counters on DUT')
+    counters = ['PORT', 'PORT_BUFFER_DROP', 'QUEUE', 'PG_WATERMARK', 'RIF']
+    for counter in counters:
+        return_value = duthost.shell("sudo redis-cli -n 4 hset 'FLEX_COUNTER_TABLE|{}' 'FLEX_COUNTER_STATUS' 'enable'".format(counter),
```

Comment:
>enable [](http://example.com/codeflow?start=110&length=6)

Instead of enable_counter, could you wait until all counters are available? Even you want to enable_counter, the wait is also useful and lead to stable test run.

Reply:
@qiluo-msft I addressed the comments to use the CLI and wait until all counters are enabled in one fixture.
can you please check?